### ISSUE TITLE
Dominos Actionbar Key Activity Blockage

### ISSUE THREAD

**Issue by OdessaGrimaldus** [tullamods/Dominos#777]
Hello, as of today there has begun being a constant "Dominos has been blocked from an action available only to the Blizzard UI" popup when using the any keys for the abilities on the action bars, not allowing them to activate.. though when the ability is clicked with the mouse cursor the error does not present itself.

**Addon Version**
Dominos-10.1.4

**World of Warcraft Version**
Retail

**Comment by AbelardTullus** [comment]
I'm encountering this issue as well, although it seems to be conflicting with other addons, oddly enough it says Bagnon is interfering with it, I've only been having this issue with Dominos but not Bartender.

A workaround I've found is using paging or a macro to switch abilities, this seems to "refresh" something and allows hotkeys to be used again.

**Comment by XimmortalAresX** [comment]
> I'm encountering this issue as well, although it seems to be conflicting with other addons, oddly enough it says Bagnon is interfering with it, I've only been having this issue with Dominos but not Bartender.
> 
> A workaround I've found is using paging or a macro to switch abilities, this seems to "refresh" something and allows hotkeys to be used again.

What specifically do you mean when you say you are using paging or a macro to switch abilities that somehow temporarily fixes the issue?

**Comment by AbelardTullus** [comment]
> What specifically do you mean when you say you are using paging or a macro to switch abilities that somehow temporarily fixes the issue?

Sorry for the late response, when I have my bar set to page (say when I hold shift, it shows a different set of buttons), the error no longer occurs. I've noticed however that sometimes this issue can occur when using macros specifically, even if said macro is simple, the macro will no longer run when I press the hotkey (although it still works when clicking on the button using the mouse). 

The error also occurs when zoning into an instance (such as dungeons/raids), regardless of whether you walked into an instance in the world or via LFD/LFR.

Examples of macros that cause the error to occur;

```
#showtooltip
/use Shadow Bolt
/petattack [exists,nodead]
```

```
#showtooltip
/use [mod:alt,known:Roll]Roll;Tiger Palm
```

**Comment by Tuller** [comment]
This should be fixed in the latest beta (10.2.5)
